docs/getting-started-with-cosmosdb.md

```diff
@@ -0,0 +1,103 @@
+## Getting Started with Scalar DB and Cosmos DB
+
+## Overview
+This document briefly explains how you can get started a simple electronic money application with scalar DB and Cosmos DB.
+
+## Install prerequisites
+
+Scalar DB is written in Java and uses Cosmos DB as an underlining storage implementation, so the following software is required to run it.
+
+* [Oracle JDK 8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) (OpenJDK 8) or higher
+* Other libraries used from the above are automatically installed through gradle
+
+### Cosmos DB setup
+
+* [Azure Cosmos DB](https://docs.microsoft.com/en-us/azure/cosmos-db/introduction)
+    * Take a look at [this document](https://docs.microsoft.com/en-us/azure/cosmos-db/create-cosmosdb-resources-portal#create-an-azure-cosmos-db-account) for how to create an Azure Cosmos DB account.
+    * Choose API as `Core (SQL)` while creating the cosmos db account.
+    * Select `Default consistency` from the left navigation on your Azure Cosmos DB account page, and then change `Consistency Level` from `SESSION` to `STRONG`.
+
+From here, we assume Oracle JDK 8 is properly installed in your local environment, and Cosmos DB is running in Azure environment.
+
+## Configure the Cosmos DB connection
+    
+The **scalardb.properties** (docs/getting-started/src/main/resources/scalardb.properties) file holds the configuration for Scalar DB. Basically, Cosmos DB account_uri and primary_key will get from Azure Cosmos DB account.
+    
+```
+# Comma separated contact points
+scalar.db.contact_points=<YOUR_ACCOUNT_URI>
+
+# Port number for all the contact points. Default port number for each database is used if empty.
+#scalar.db.contact_port=
+
+# Credential information to access the database
+scalar.db.username=
+scalar.db.password=<YOUR_ACCOUNT_PASSWORD>
+
+# Storage implementation. Either cassandra or cosmos can be set. Default storage is cassandra.
+scalar.db.storage=cosmos
+```
+
+## Build
+
+Please use [this](getting-started-with-cassandra.md#Build) to create a build.
+
+## Set up database schema
```

Comment:
I think from here, most of the sentences are copies from the Cassandra doc.
So,  let's remove the redundancy and state only the differences.

Reply:
I think it is needed here because `Set up database schema` contains changes.
The schema and execution command are different.

Do you expect a single document with both conditions (Cassandra and cosmos db)?